catalyst/contrib/callbacks/draw_masks_callback.py

```diff
@@ -0,0 +1,205 @@
+from typing import Iterable, List, Optional, TYPE_CHECKING
+import os
+
+import numpy as np
+from skimage.color import label2rgb
+from skimage.color.colorlabel import DEFAULT_COLORS
+
+import torch
+
+from catalyst import utils
+from catalyst.callbacks import ILoggerCallback
+from catalyst.contrib.tools.tensorboard import SummaryWriter
+from catalyst.contrib.utils.cv.tensor import tensor_to_ndimage
+from catalyst.core.callback import CallbackNode, CallbackOrder
+
+if TYPE_CHECKING:
+    from catalyst.core.runner import IRunner
+
+DEFAULT_COLORS = np.array(DEFAULT_COLORS)
+
+
+class DrawMasksCallback(ILoggerCallback):
+    """
+    Logger callback draw masks for common segmentation task: image -> masks
+    """
+
+    def __init__(
+        self,
+        pred_mask_key: str,
+        image_key: Optional[str] = None,
+        gt_mask_key: Optional[str] = None,
```

Comment:
is it correct, that
- `pred_mask_key ` comes from `runner.output`
- `image_key` comes from `runner.input`
- `gt_mask_key ` comes from `runner.input`
?

could we rename them into
- output_key (or maybe even output_logits_key)
- input_image_key
- input_mask_key
?

what do you think about it?

Reply:
Yes it is correct. predicted mask(logits) we get from "runner.output", image(input) and gt_mask(target) we get from "runner.input". The names are all debatable, if you want other names, I can change them.

Reply:
could you please rename them?
it would be great if all our callbacks will follow the same naming